src/components/EmptyState/EmptyState.tsx

```diff
@@ -29,105 +30,113 @@ export interface EmptyStateProps {
   footerContent?: React.ReactNode;
 }
 
-export class EmptyState extends React.PureComponent<EmptyStateProps, never> {
-  render() {
-    const {
-      children,
-      heading,
-      image,
-      largeImage,
-      imageContained,
-      action,
-      secondaryAction,
-      footerContent,
-    } = this.props;
+export function EmptyState({
+  children,
+  heading,
+  image,
+  largeImage,
+  imageContained,
+  action,
+  secondaryAction,
+  footerContent,
+}: EmptyStateProps) {
+  return (
+    <WithinContentContext.Consumer>
```

Comment:
As you're in a functional component you can use hooks instead of the render prop function

```jsx
const withinContentContainer = useContext(WithinContentContext);

/* stuff using withinContentContainer */
```

instead of 

```jsx
return (
  <WithinContentContext.Consumer>
    {(withinContentContainer) => {
      /* stuff using withinContentContainer */
    }}
  </WithinContentContext.Consumer>
);

Reply:
Gotcha, thanks!